arc_solvers/training_config/qa/multi_choice/reader_qa_multi_choice_max_att_ARC_Chellenge_full.json

```diff
@@ -0,0 +1,86 @@
+{
+  "dataset_reader": {
+    "type": "arc-multi-choice-json",
+    "token_indexers": {
+        "tokens": {
+            "type": "single_id",
+            "lowercase_tokens": true
+        }
+    },
+    "tokenizers": {
+        "default": {
+            "start_tokens": ["@start@"],
+            "end_tokens": ["@end@"]
+        },
+    },
+  },
+  "train_data_path": "../data/allenai/ARC-V1-Feb2018/ARC-Challenge/ARC-Challenge-Train.jsonl",
```

Comment:
Are these paths accurate in the context of this repo ? Shouldn't it be `data/ARC-V1-Feb2018/ARC-Challenge/ARC-Challenge-Train.jsonl`

Reply:
This is the correct path as the run.py need to be executed in the arc_solvers/run.py and data folder is in the root path of the project.

Reply:
Even then, why is there `allenai` in the path ? Also generally, you would run it as `python arc_solvers/run.py` i.e. from the root directory.